### DISCUSSION TITLE
Support for recurring tasks

### DISCUSSION THREAD

**Discussion by arunnbabu81** [NicolasPetton/Projectify#11] [Ideas]
Can recurring tasks be supported in projectify? I could find only this as reference for a similar implentation. 

1. [ticklers plugin](https://btheado.github.io/tw-ftlist/#ticklers%20plugin)  
2. [google groups discussion](https://groups.google.com/g/tiddlywiki/c/9oXj7sjzT-8/m/9HPX7W86CgAJ) 
3. [google groups discussion](https://groups.google.com/g/tiddlywiki/c/QKL-s1MSHT0/m/fuUwz_lhFgAJ)
4. [Cardo](https://dyumnin.com/Cardo.html) [google groups discussion](https://groups.google.com/u/1/g/tiddlywiki-gtd/c/Te0fYV8OW44)

**Comment by NicolasPetton** [comment]
Yes, definitely. It's in the roadmap, I want to add support for recurring tasks soon.
